samples/monster_generated.h

```diff
@@ -185,6 +191,9 @@ struct MonsterT : public flatbuffers::NativeTable {
 
 struct Monster FLATBUFFERS_FINAL_CLASS : private flatbuffers::Table {
   typedef MonsterT NativeTableType;
+  static flatbuffers::TypeTable* DefaultTypeTable() {
```

Comment:
Please make sure the `*` goes next to the name. Also, `Default` doesn't sound quite right, since there are no alternatives. Just `TypeTable` would be too prone to clashes. Maybe.. `ThisTypeTable` ? `GetTypeTable` ? `TheTypeTable` ? Or go verbose and say `MiniReflectTypeTable`..?

Reply:
Sounds good, will submit fix. Thanks!